charts/retool/templates/_helpers.tpl

```diff
@@ -52,13 +52,67 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
 {{/*
-Selector labels
+Selector labels for main backend. Note changes here will require deployment
+recreation and incur downtime. The "app.kubernetes.io/instance" label should
+also be included in all deployments, so telemetry knows how to find logs. 
 */}}
 {{- define "retool.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "retool.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
+{{/*
+Selector labels for workflow backend. Note changes here will require manual
+deployment recreation and incur downtime, so should be avoided.
+*/}}
+{{- define "retool.workflowBackend.selectorLabels" -}}
+retoolService: {{ template "retool.workflowBackend.name" . }}
+{{- end }}
```

Comment:
This and all the `retool.*.selectorLabels` partials below don't actually change the selector labels that were already used, they're just refactoring the template code.